### PR TITLE
Update 'Administer Custom Groups' to show subtypes

### PR DESCRIPTION
**Pull request by mattwire**
Overview
----------------------------------------
Go to "Administer Custom Groups".

Before
----------------------------------------
Does not show custom group subtypes.

After
----------------------------------------
Shows custom group subtypes.

Technical Details
----------------------------------------

Comments
----------------------------------------

### CODE REVIEW

File: ext/civicrm_admin_ui/managed/SavedSearch_Administer_Custom_Groups.mgd.php [hunk 1]
```diff
@@ -135,10 +143,11 @@
                   'join' => '',
                   'target' => '',
                   'icon' => 'fa-list-alt',
-                  'text' => E::ts('Fields (%1)', [1 => '[COUNT_CustomGroup_CustomField_custom_group_id_01_id]']),
+                  'text' => E::ts('Fields ([COUNT_CustomGroup_CustomField_custom_group_id_01_id])'),
```
Comment (colemanw) [inline ext/civicrm_admin_ui/managed/SavedSearch_Administer_Custom_Groups.mgd.php:146]: Can you revert this line

Comment (mattwire) [reply ext/civicrm_admin_ui/managed/SavedSearch_Administer_Custom_Groups.mgd.php]: Done

**Comment by mattwire** [comment]
This PR solves the issue of Activity subtypes not showing in the UI. However, we need to check that other subtypes (eg. Participant, Event etc. work).  Ideal for someone to test at a sprint :-)

**Comment by herbdool** [comment]
Looks good to me! I created custom groups for a couple different entity subtypes:

2024-03-02 15 38 08 core-29584-8h9sa test-1 civicrm org eb79d59e7a6d

**Comment by herbdool** [comment]
@mattwire looks ready to me.

**Pull request merged**